src/pybind/mgr/dashboard/rgw_detail.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<script>
+        $(document).ready(function(){
+            // Pre-populated initial data at page load
+            var content_data = {{ content_data }};
+
+            var refresh = function() {
+                $.get("{{ url_prefix }}/rgw/rgw_data/" + content_data.rgw_id + "/", function(data) {
+                   content_data = data;
+                   setTimeout(refresh, 3000);
+                });
+            };
+            setTimeout(refresh, 5000);
+
+            var post_load = function() {
+                content_data.rgw_metadata_list = [];
+                content_data.rgw_status_list = [];
+                _.each(content_data.rgw_metadata, function(v, k) {
```

Comment:
It would also be fine to do this on the python side, so that the javascript just gets the data in the form that it prefers (a list).

These items are currently coming out in arbitrary order, which makes it a bit difficult to scan visually -- it would be nice to sort them alphabetically by key (could also do this server side if passing a list instead of a dict)

If you choose to keep it on the javascript side, you'll also need to make sure that post_load is called after each `refresh()`